<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>523</width>
             <height>1218</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QFrame" name="frame">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>1200</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <widget class="QLabel" name="label">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>10</y>
                 <width>81</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>MENU</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>160</x>
                 <y>60</y>
                 <width>171</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Traditional Meals</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_ACHU">
               <property name="geometry">
                <rect>
                 <x>26</x>
                 <y>100</y>
                 <width>191</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_3">
               <property name="geometry">
                <rect>
                 <x>250</x>
                 <y>140</y>
                 <width>101</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>2000 FCFA</string>
               </property>
              </widget>
              <widget class="QPushButton" name="Add_Achu">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>140</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Add to Card</string>
               </property>
              </widget>
              <widget class="QPushButton" name="Add_Eru">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>300</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Add to Card</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_ERU">
               <property name="geometry">
                <rect>
                 <x>26</x>
                 <y>260</y>
                 <width>191</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_4">
               <property name="geometry">
                <rect>
                 <x>250</x>
                 <y>300</y>
                 <width>101</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>1000 FCFA</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_5">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>620</y>
                 <width>141</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Western Meals</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>250</x>
                 <y>720</y>
                 <width>101</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>2000 FCFA</string>
               </property>
              </widget>
              <widget class="QPushButton" name="Add_pasta">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>720</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Add to Card</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_Pasta">
               <property name="geometry">
                <rect>
                 <x>26</x>
                 <y>680</y>
                 <width>191</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_7">
               <property name="geometry">
                <rect>
                 <x>250</x>
                 <y>450</y>
                 <width>101</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>1200 FCFA</string>
               </property>
              </widget>
              <widget class="QPushButton" name="Add_koki">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>450</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Add to Card</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_Koki">
               <property name="geometry">
                <rect>
                 <x>26</x>
                 <y>410</y>
                 <width>191</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QPushButton" name="Add_frid">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>870</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Add to Card</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_Frid">
               <property name="geometry">
                <rect>
                 <x>26</x>
                 <y>830</y>
                 <width>191</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_8">
               <property name="geometry">
                <rect>
                 <x>250</x>
                 <y>870</y>
                 <width>101</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>2500 FCFA</string>
               </property>
              </widget>
              <widget class="QPushButton" name="Add_meat">
               <property name="geometry">
                <rect>
                 <x>370</x>
                 <y>1020</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>Add to Card</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_Meat">
               <property name="geometry">
                <rect>
                 <x>26</x>
                 <y>980</y>
                 <width>191</width>
                 <height>131</height>
                </rect>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
              <widget class="QLabel" name="label_9">
               <property name="geometry">
                <rect>
                 <x>250</x>
                 <y>1020</y>
                 <width>101</width>
                 <height>31</height>
                </rect>
               </property>
               <property name="font">
                <font>
                 <pointsize>14</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>8000 FCFA</string>
               </property>
              </widget>
              <widget class="QPushButton" name="View_card">
               <property name="geometry">
                <rect>
                 <x>220</x>
                 <y>1150</y>
                 <width>121</width>
                 <height>41</height>
                </rect>
               </property>
               <property name="text">
                <string>View Card</string>
               </property>
              </widget>
              <widget class="QPushButton" name="order_history">
               <property name="geometry">
                <rect>
                 <x>410</x>
                 <y>10</y>
                 <width>80</width>
                 <height>21</height>
                </rect>
               </property>
               <property name="text">
                <string>order history</string>
               </property>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QPushButton" name="pay">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>230</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>PAY</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>230</y>
          <width>151</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Orange Money</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>260</y>
          <width>131</width>
          <height>19</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>MTN Money</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>230</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>TEL:</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="ViewOder_text">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>371</width>
          <height>181</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>13</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ORDER:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>423</x>
          <y>250</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QPushButton" name="Back">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>260</y>
          <width>80</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>0</y>
          <width>51</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>History</string>
        </property>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>45</x>
          <y>31</y>
          <width>451</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>575</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
